client/classes/ApplicationState.js

```diff
@@ -253,6 +267,49 @@ define(
                 swal('Error controlling agent.');
             };
 
+            var countdown = function() {
+                // decrement the progress integer. progress starts as the average start time of the app
+                newsec = self.progress();
+                newsec--;
+                self.progress(newsec)
+            };
+
+            self.startCountdown = function(){
+                var avgStartTime = parseInt(self.loadTimes.ave);
+                if (avgStartTime === 0) {
+                    self.progress('Not enough data');
+                    return
+                }
+                // bake in 5s to run the methods on sentinel/get update from Zoom
+                var adjustment = 5;
+                if (self.lastUpdate() !== "") {
+                    var last = Date.parse(self.lastUpdate());
+                    var now = new Date();
+                    timeFromUpdate = parseInt(((now.getTime() - last))/ 1000)
+                }
+                // if you load the page n seconds after a start, decrement n seconds from the progress number
+                var sec = avgStartTime - timeFromUpdate + adjustment;
+                
+                self.progress(sec);
+                var cdID = setInterval(countdown, 1000);
+                // keep track of this id so we can stop the countdown later
+                self.countdownid(cdID)
+            };
+
+            self._startCountdownAtRestart = ko.computed(function() {
+                // only show when admin is enabled
```

Comment:
I like this, at least initially. Do we know if during startup people typically have admin enabled?